README.md

```diff
@@ -20,6 +20,7 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
 
 ## Python
 - [F-Droid Server](https://gitlab.com/fdroid/fdroidserver/) - Tools for managing builds, indexes, updates, and deployments for F-Droid repositories.
+- [PhishBuster](https://github.com/VFXGamer/PhishBuster) - A Phishing Site Detection Tool (Web Based)
```

Comment:
Please add some Github Issues so people know what to contribute to.

Also, this repo is missing the hacktoberfest repo topic.

Reply:
What do you mean by "missing the hacktoberfest repo topic" this is my first time as a maintainer.

Reply:
Maybe I misread earlier, I'm seeing it now: 
![image](https://user-images.githubusercontent.com/1885159/135715887-ccae520b-00ef-4a1b-9db0-b448cc78125b.png)

So approving this :)